docs/design/batch-request.md

```diff
@@ -0,0 +1,260 @@
+# Batch Requests
+
+
+## Overview
+
+Batch requests consolidate multiple API calls within a single HTTP request. Clients bundle multiple requests into an array and receive an array of responses in return. The server may process these batched requests in any sequence, concurrently. Clients must then correlate the responses to the original requests using the unique ID of each object.
+
+This is a standard JSON-RPC 2.0 feature. For more information, see the [JSON-RPC 2.0 Specification - Batch](https://www.jsonrpc.org/specification#batch)
+
+## Goals / Motivation
+1. Adhere to the JSON-RPC 2.0 specification for batch requests
+2. Support Tooling that uses batch requests, such as EthersJs v6
+3. Support batch requests for all methods that are supported for single requests
```

Comment:
Note that certain methods aren't supported for batch e.g. debug and trace.
We should probably add a config that captures a list of methods not supported.

Reply:
Yes, I noticed that on the documentation for `Alchemy` but did not found any exceptions (or way to configure that) on the documentation for ethersjs, but I agree that we should configure them via property. will add it to the design doc and add a follow up ticket for that.

Reply:
https://github.com/hashgraph/hedera-json-rpc-relay/issues/1885

Reply:
✅  added to Documentation and created a follow up ticket for this #1885